ROSNET/Type/PrimitiveType.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace ROSNET.Type
+{
+    /// <summary>
+    /// Enum-class used for datatype attribute in FieldValue-class
+    /// </summary>
+    public enum PrimitiveType
+    {
+        BOOL, INT8, UINT8, INT16, UINT16, INT32,
```

Comment:
https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions (also valid for `ROSTNET`)

Reply:
Det er nok min feil her ja... Enig at disse må formateres litt

Reply:
Riktig format her blir da "Bool", "Int8" etc?